Include the list of key bindings (shortcuts) in the documentation (#1805)
sublimelsp/LSP: docs/src/features.md

> 

Author: This page should probably only have a high-level overview of the main features with screencasts. The existing content doesn't feel ideal here.

## Commands and shortcuts

> Apart from the commands listed in the [Keyboard Shortcuts](keyboard_shortcuts.md) section, LSP also provides more generic commands in the Command Palette:

Reviewer: ```suggestion
LSP provides the following commands in the Command Palette:
```

The beginning of the sentence is maybe unnecessary, because you also mention the keyboard shorcuts in the bullet point bellow:
`Preferences: LSP Key Bindings`: opens LSP key bindings configuration - see [Keyboard Shortcuts](keyboard_shortcuts.md)`

Author: Yes but the point was that those listed below are not the only ones that LSP provides.
But I don't have very strong feeling about that either way.

Reviewer: I also do not have a preference here, it just felt a bit redundant.
